tests/backup/backup_orphaned_delete_test.go

```diff
@@ -0,0 +1,505 @@
+				if err != nil {
+					restoreUID, UIDError := Inst().Backup.GetRestoreUID(nonAdminCtx, restoreName, orgID)
+					log.FailOnError(UIDError, "failed to get restore %s uid", restoreName)
+					restoreInspectRequest := &api.RestoreInspectRequest{
+						Name:  restoreName,
+						OrgId: orgID,
+						Uid:   restoreUID,
+					}
+					resp, inspectError := Inst().Backup.InspectRestore(nonAdminCtx, restoreInspectRequest)
+					log.FailOnError(inspectError, "failed to inspect restore %s", backupName)
+					if resp.GetRestore().GetStatus().GetStatus() != api.RestoreInfo_StatusInfo_Failed {
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of restore %s of backup %s", restoreName, backupName))
+					}
+				}
+				mu.Lock()
+				defer mu.Unlock()
+				userFailedRestoreNames = append(userFailedRestoreNames, restoreName)
+			}
+			for backupName, namespace := range userSuccessfulBackupMap {
+				wg.Add(1)
+				restoreName := fmt.Sprintf("%s-%s", restoreNamePrefix, backupName)
+				go createRestore(backupName, restoreName, namespace)
+			}
+			wg.Wait()
+			log.Infof("The list of failed user restores taken are: %v", userFailedRestoreNames)
+		})
+		Step(fmt.Sprintf("Verify failed restores of the user %s from the admin", infraAdminUser), func() {
+			log.InfoD(fmt.Sprintf("Verifying failed restores of the user %s from the admin", infraAdminUser))
+			nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+			log.FailOnError(err, "failed to fetch user %s ctx", infraAdminUser)
+			userOwnerID, err := portworx.GetSubFromCtx(nonAdminCtx)
+			log.FailOnError(err, "failed to fetch user owner id %s", infraAdminUser)
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			restoreNamesByOwnerID, err := GetAllRestoreNamesByOwnerID(userOwnerID, orgID, ctx)
+			log.FailOnError(err, "failed to fetch restore names with owner id %s from the admin", userOwnerID)
+			for _, restoreName := range userFailedRestoreNames {
+				if !IsPresent(restoreNamesByOwnerID, restoreName) {
+					err := fmt.Errorf("failed restore %s is not listed in restore names %s", restoreName, restoreNamesByOwnerID)
+					log.FailOnError(fmt.Errorf(""), err.Error())
+				}
+			}
+		})
+		Step(fmt.Sprintf("Delete failed restores taken by user %s from the admin", infraAdminUser), func() {
+			log.InfoD(fmt.Sprintf("Deleting failed restores taken by user %s from the admin", infraAdminUser))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, restoreName := range userFailedRestoreNames {
+				restoreUID, err := Inst().Backup.GetRestoreUID(ctx, restoreName, orgID)
+				log.FailOnError(err, "failed to fetch restore %s uid", restoreName)
+				err = DeleteRestoreWithUID(restoreName, restoreUID, orgID, ctx)
+				log.FailOnError(err, "failed to delete restore %s uid", restoreName)
+			}
```

Comment:
same.
Can we combine the two steps

Reply:
In TestRail these are two different steps.